config.go

```diff
@@ -260,6 +260,12 @@ type Config struct {
 	// QueueCheckInterval is the interval at which we check the message
 	// queue to apply the warning and max depth.
 	QueueCheckInterval time.Duration
+
+	// MsgpackUseNewTimeFormat when set to true, force the underlying msgpack
+	// codec to use the new format of time.Time when encoding (used in
+	// go-msgpack v1.1.5 by default). Decoding is not affected, as all
+	// go-msgpack v2.1.0+ decoders know how to decode both formats.
+	MsgpackUseNewTimeFormat bool
```

Comment:
I'm not sure I fully understand, shouldn't this comment be: 
 ```suggestion
	// MsgpackUseNewTimeFormat when set to false, force the underlying msgpack
	// codec to use the old format of time.Time when encoding (used in
	// go-msgpack v1.1.5 by default). Decoding is not affected, as all
	// go-msgpack v2.1.0+ decoders know how to decode both formats.
	MsgpackUseNewTimeFormat bool
```

Reply:
I believe those statements are equivalent.

The default is `false` though, so I went with documenting what happens when you change the value to true. (I have nearly the same comment everywhere else, with the same parameter name.)